python/lsst/ts/linearstage/controllers/zaber_lst.py

```diff
@@ -19,23 +19,229 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-__all__ = ["Zaber"]
+__all__ = ["Zaber", "ZaberV2"]
 
 import asyncio
+import logging
 import os
 import pty
+import types
 
 import yaml
-from lsst.ts.linearstage.mocks.mock_zaber_lst import MockSerial
+from lsst.ts import salobj
+from lsst.ts.linearstage.mocks.mock_zaber_lst import LinearStageServer, MockSerial
 from zaber import serial as zaber
 
+try:
+    from zaber_motion import Units
+    from zaber_motion.ascii import AxisType, Connection, Device
+    from zaber_motion.exceptions import (
+        CommandFailedException,
+        ConnectionFailedException,
+    )
+except ImportError:
+    pass
```

Comment:
What is supposed to happen if the import fails? Please at least add an inline comment here that explains that and why the except block is empty.